### ISSUE TITLE
Remove automate_the_scraping.sh

### ISSUE THREAD

**Issue by cornzyblack** [Cherrymoon0/Automate_Python_Script_for_scraping#2]
What does `automate_the_scraping.sh` script do? Do you plan on using it for the scraping project?
@Cherrymoon0  @kldplace

**Comment by kldplace** [comment]
yes, we plan to use `automate_the_scraping.sh` script for scraping project to automate our scrape to meke run automatically once a day.
